packages/shared/src/components/cards/RaisedLabel.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import styles from './Card.module.css';
+import { SimpleTooltip } from '../tooltips/SimpleTooltip';
+
+export function RaisedLabel({
+  trending,
+  listMode,
+  type,
+}: {
+  trending: number;
+  listMode?: boolean;
+  type: 'hot' | 'pinned';
```

Comment:
I'd suggest making this a enum in this case just to be more explicit.
See for instance ModalFooter Justify function.